pysteps/feature/tstorm.py

```diff
@@ -235,6 +250,7 @@ def get_profile(areas, binary, ref, loc_max, time, minref):
         cells_id.cen_x.iloc[n] = int(np.nanmean(cells_id.x[n]))  # int(x[0])
         cells_id.cen_y.iloc[n] = int(np.nanmean(cells_id.y[n]))  # int(y[0])
         cells_id.max_ref.iloc[n] = maxref
+        cells_id.area.iloc[n] = len(cells_id.x.iloc[n])
```

Comment:
is the area approximated by the number of x coordinates, or am I getting this code line wrong?

Reply:
Yes, the cell area is approximated by the number of pixels (i.e. the number of x- or y-coordinates) having the cell label. Is this OK?

Reply:
Uh, right, very good, thanks!